### PR TITLE
Add background for empty screen

### PR DESCRIPTION
**Pull request by rsanchez15**
Signed-off-by: RaulSanchez <raul@eprosima.com>

### CODE REVIEW

File: qml.qrc [hunk 1]
```diff
@@ -63,6 +63,8 @@
         <file alias="eprosima_logo_horizontal.png">resources/images/eprosima_logo_horizontal.png</file>
         <file alias="eprosima_logo.ico">resources/images/eprosima_logo.ico</file>
         <file alias="fastdds-monitor-logo.png">resources/images/fastdds-monitor-logo.png</file>
+        <file alias="monitor_by_eprosima.svg">resources/images/monitor_by_eprosima.svg</file>
```
Comment (juanlofer-eprosima) [inline qml.qrc:66]: What is the purpose of this file?

File: qml/ChartsLayout.qml [hunk 2]
```diff
@@ -230,6 +230,96 @@ Rectangle {
...
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    fontSizeMode: Text.Fit
+                    minimumPixelSize: 10
+                    font.pointSize: 20
+                    font.bold: true
+                    color: Theme.x11Grey
+                    text: mainApplicationView.monitors == 0 ? "Ups... no initilized monitors yet"
```
Comment (juanlofer-eprosima) [inline qml/ChartsLayout.qml:279]:
```suggestion
                    text: mainApplicationView.monitors == 0 ? "Oops... no initialized monitors yet"
```

Comment (juanlofer-eprosima) [reply qml/ChartsLayout.qml:279]: I would not write "Oops" as nothing went wrong.

File: qml/ChartsLayout.qml [hunk 3]
```diff
@@ -230,6 +230,96 @@ Rectangle {
...
+                                                            : "Ups... no data charts yet"
```
Comment (juanlofer-eprosima) [inline qml/ChartsLayout.qml:280]:
```suggestion
                                                            : "Oops... no data charts yet available"
```

Comment (juanlofer-eprosima) [reply qml/ChartsLayout.qml:280]:
```suggestion
                                                            : "Oops... no data charts to display"
```

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]
Please fix uncrustify test

File: qml.qrc (hunk 1 above)
Comment (rsanchez15) [reply qml.qrc:66]: That was another design for the background. I'd remove it as it is no longer needed

File: qml/ChartsLayout.qml (hunk 2 above)
Comment (rsanchez15) [reply qml/ChartsLayout.qml]: It seems that the "oops" will stay

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**